2.cellprofiler_analysis/README.md

```diff
@@ -0,0 +1,16 @@
+# Segmentation and feature extraction using CellProfiler
+
+In this module, we segment nuclei, cytoplasm, and whole cells from the images and perform feature extraction using CellProfiler.
+
+## Run the `nf1_analysis` notebook
+
+To run the CellProfiler analysis pipeline on the illumination corrected images for each plate, run the [nf1_analysis.ipynb](nf1_analysis.ipynb) notebook as a python script using the code block below:
+
+```bash
+# Run this script in terminal
+# move to the 2.cellprofiler_analysis directory to access the `sh` script
+cd 2.cellprofiler_analysis
```

Comment:
What would happen if one didn't change directory? Could one just prepend the directory to the command below?

Reply:
I tried adding it into the sh file, and it wouldn't work because it couldn't find the file. So if they tried running this without changing the directory, it just wouldn't find the file and give a file not found error